templates/welcome.html

```diff
@@ -0,0 +1,6 @@
+<html>
+<body>
+  <h1>Добро пожаловать, {{ name }}!</h1>
+  <a href="/logout">Выйти</a>

```

Comment:
А чтобы новая функция `welcome` работала нужно сделать так
```suggestion
<form method="post">
    <button type="sumbit">Выйти</button>
</form>
```